setup.py

```diff
@@ -40,6 +40,7 @@ def import_requirements():
     maintainer='Ensembl Production Team',
     maintainer_email='ensembl-production@ebi.ac.uk',
     python_requires='>=3.8',
+    install_requires=import_requirements(),
```

Comment:
can we add CI_COMMIT_TAG to version similar to other apps 
version=os.getenv('CI_COMMIT_TAG', version),

Reply:
Well spotted! just updated the `setup.py` file accordingly.